PatelDLTGroupStamps/Views/StampStockView.swift

```diff
@@ -8,27 +8,79 @@
 import SwiftUI
 
 struct StampStockView: View {
+
+  @StateObject private var stampHelper = StampHelper()
+  @State private var shoppingAlert = false
   
   var body: some View {
     
     // MARK: Assignment 2: Summary screen and detail screen
-    
     NavigationStack {
-      List {
-        ForEach(stamp.unitedNationsStamps, id: \.self) {
-          unitedNationStamp in
-          NavigationLink(unitedNationStamp.description, destination: StampView(stamp: unitedNationStamp))
+      List(stampHelper.stamps, id: \.id) { stamp in
```

Comment:
A little bit unrelated, but you are running into the double navigation bar issue again here. Consider what kind of stack is embedded in your view here.